src/main/java/Task.java

```diff
@@ -0,0 +1,13 @@
+public abstract class Task {
+
+    boolean checked = false;
```

Comment:
boolean should be named "isChecked" instead, according to Java Coding Standard 👍